server/methods/addAllUserToRoom.js

```diff
@@ -1,7 +1,8 @@
 Meteor.methods({
-	addAllUserToRoom(rid) {
+	addAllUserToRoom(rid, activeUsersOnly) {
```

Comment:
What about changing this so that it defaults to `false` making it an optional parameter with a default value? Something like: `activeUsersOnly = false`